jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/AccelerationSet4WayBuilder.java

```diff
@@ -41,7 +41,7 @@ public static J2735AccelerationSet4Way genericAccelerationSet4Way(JsonNode accel
       // -- the value -2000 shall be used for values less than -2000
       // -- a value of 2001 shall be used for Unavailable
       if (accelLong == 2001) {
-         genericAccelerationSet4Way.setAccelLong(null);
+         genericAccelerationSet4Way.setAccelLong(BigDecimal.valueOf(2001));
```

Comment:
Do we have unit test to cover all the updated unavailable values?

Reply:
Yes, the unit tests all pass for these new values. We technically have existing tests for these but they were expecting null outputs. I had to alter these to expect the new unavailable values instead. I renamed the tests to testUnavailable instead of testNull like they were previously named. I made additional tests to test actual null values.